index.html

```diff
@@ -47,11 +49,47 @@
           <div class="drop-bottom">
             <hr>
           </div>
-       
-       
-        
- </div>
+          </div>
+     
+          
+            <div class="popup2"  >
+              <h1 id="mob-pop"  >Multi post Stories</h1>
+              <h2 id="desk-pop" >Keeping track of hundreds  of components website</h2>
+             <div class="close-button-container2">
+               <button type="button"  class="close-button2" onclick="closePopup()"> &times;</button>
+             </div>
+             
+             <ul class="cardbtn">
+               <li><button type="button" class="middle-pop">html</button></li>
+               <li> <button type="button" class="middle-pop">bootstrap</button></li>
+               <li><button type="button" class="middle-pop">Ruby On Rails</button></li>
```

Comment:
- [ ] the technologies used should not be buttons, kindly put the text directly inside the `li`element